src/python/Utils/MemoryCache.py

```diff
@@ -20,10 +18,10 @@
 
 class MemoryCacheException(Exception):
     def __init__(self, message):
-        super(MemoryCacheException, self).__init__(message)
-
+        #super(MemoryCacheException, self).__init__(message)
```

Comment:
Andrea, was there any specific reason for this removal? If it was a pylint warning, then I think it's actually suggesting to use the latest "style" for this call, which I think it would be: `super().__init__(message)`.